### PR TITLE
Updating name of container to inject proxy information into

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: manifests/07-operator.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ kind: Deployment
 metadata:
   name: cluster-image-registry-operator
   namespace: openshift-image-registry
-  annotations:
+  labels:
```
Comment (bparees) [inline manifests/07-operator.yaml]: this isn't what changed.  the configmap changed from annotation to label.  The env vars are still injected via annotation on the deployment.

File: manifests/07-operator.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ kind: Deployment
...
     config.openshift.io/inject-proxy: operator
```
Comment (bparees) [inline manifests/07-operator.yaml]: but this value is wrong.  it's supposed to be the name of the container you want injected.  your container is named cluster-image-registry-operator

Comment (coreydaley) [reply manifests/07-operator.yaml]: Updated.  Interesting ... Can you point me to some documentation on that?  I looked all over when I set this up and only found another operator that was doing it and theirs just said operator like this one did.

**Comment by coreydaley** [comment]
@bparees ptal, updated per your comment

File: manifests/07-operator.yaml (hunk 2 above)
Comment (bparees) [reply manifests/07-operator.yaml]: @coreydaley https://github.com/openshift/cluster-version-operator/pull/224#issue-298946446

i dunno that it's documented anywhere unfortunately. 

@danehans we need to make sure teams that have added this label to their operator deployment did so correctly.  the value of the label needs to be the name(s) of the container(s) that are to get the env var injection.

**Pull request merged**